plugins/chat/assets/javascripts/discourse/components/styleguide/chat-thread-list-item.js

```diff
@@ -1,10 +1,19 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
 import { getOwner } from "@ember/owner";
+import { next } from "@ember/runloop";
 import { service } from "@ember/service";
 import ChatFabricators from "discourse/plugins/chat/discourse/lib/fabricators";
 
 export default class ChatStyleguideChatThreadListItem extends Component {
   @service currentUser;
+  @tracked thread;
 
-  thread = new ChatFabricators(getOwner(this)).thread();
+  constructor() {
+    super(...arguments);
+
+    next(() => {
```

Comment:
Assigning `this.thread`directly in the constructor was triggering the following error:

```
Assertion Failed: You attempted to update `participantUsers` on `ChatThreadPreview`, but it had already been used previously in the same computation.  Attempting to update a value after using it in a computation can cause logical errors, infinite revalidation bugs, and performance issues, and is not supported.
```

This is a workaround.